nc_test4/tst_parallel_zlib.c

```diff
@@ -60,6 +60,10 @@ main(int argc, char **argv)
     /*printf("mpi_name: %s size: %d rank: %d\n", mpi_name,
       mpi_size, mpi_rank);*/
 
+    /* This program must have exactly 4 processors. */
+    if (mpi_size != 4)
```

Comment:
Re-running checks, I suspect (without looking) that the 'failure' was a timeout from Github Actions.  I may also add a `printf()` statement here to avoid a issue in the future where this is failing with mpi_size != 4 but that cause is not immediately obvious. Thanks!

Reply:
The CI failed because of the DAP remote3 test in the CMake build. If you merge my other open PR, which turns off DAP remote testing in the CMake build, then rerun these tests, they will pass.

A printf is not needed when the ERR macro is used. It prints an error message that gives the line number, and then I go to the source and find the exact line that failed.

Reply:
Fair enough for us and other devs, but we get a lot of support questions from end users who send in the error message for us to dig in to.  A decent subset of these are people who are running tests/files outside of the normal 'make check/make test' workflow.  Encouraging that folk are curious and poking around.  Anyways, adding the explicit message (which I've got in a separate branch I'll add, nothing needs change here) should at least hopefully circumvent any of those questions.  Thanks!